sdk/spring/spring-cloud-azure-stream-binder-eventhubs/src/test/java/com/azure/spring/cloud/stream/binder/eventhubs/config/EventHubsBinderConfigurationTests.java

```diff
@@ -63,4 +95,146 @@ void shouldConfigureArmChannelProvisionerWhenResourceManagerProvided() {
                 assertThat(channelProvisioner).isInstanceOf(EventHubsChannelResourceManagerProvisioner.class);
             });
     }
+
+    @Test
+    void shouldConfigureConsumerPrefetchCount() {
+        new ApplicationContextRunner()
+            .withConfiguration(AutoConfigurations.of(TestProcessorContainerConfiguration.class))
+            .withPropertyValues(
+                "spring.cloud.stream.eventhubs.bindings.consume-in-0.consumer.prefetch-count=150",
+                "spring.cloud.stream.eventhubs.bindings.consume-in-0.consumer.destination=dest",
+                "spring.cloud.stream.eventhubs.bindings.consume-in-0.consumer.group=group",
+                "spring.cloud.stream.eventhubs.namespace=namespace"
+                )
+            .run(context -> {
+                EventHubsExtendedBindingProperties properties = context.getBean(EventHubsExtendedBindingProperties.class);
```

Comment:
so  complicated?

Reply:
Yes, too complex, I don't find a better way to replace the mock builder.